README.md

```diff
@@ -605,6 +605,92 @@ graves errores).
+cumple los propositos de sus usuarios. La complejidad en la cual estamos
+interesados es la de este propósito y esto es lo que tratamos de clasificar como
+accidental o esencial. Hasta ahora vemos la complejidad esencial como "la
+complejidad, la cual, el equipo *tendrá* que prestarle atención, incluso en un
+mundo ideal".
+
+Mencionar que el "tener que" partir de esta observación es fundamental, si hay
+cualquier forma *posible* de que el equipo pudiera producir un sistema que los
+usuarios consideraran correcto *sin* tener en consideración el tipo de
+complejidad dada, entonces esa complejidad no sería esencial.
+
+Teniendo en cuenta que en un mundo ideal no todas las *posibles* opciones son
+válidas, la conclusión es que ningún desarrollador de verdad necesitará lidiar
+con *algo* de complejidad accidental. La definición no pretende negar esto,
+simplemente identificar su naturaleza secundaria.
+
+En última instancia (como veremos más adelante en la sección 7) nuestra
+definición es equivalente a decir que lo que es esencial para el equipo es lo
+que a los *usuarios* les preocupa. Esto es porque en un mundo ideal estaríamos
+usando un lenguaje o infrastructura la cual nos dejaría expresar directamente
+los problemas de los usuarios sin tener que expresar nada más, y así es como
+llegamos a las definiciones dadas anteriormente.
+
+El argumento podría ser presentado como que, en un mundo *ideal* podríamos
+encontrar una infrastructura que ya resuelva el problema de los usuarios por
+completo. Si bien, es posible imaginar que alguien ya ha hecho el trabajo, no es
+particularmente esclarecedor, puede ser mejor considerar una restricción
+implícita de que el hipotético lenguaje e infrastructura sean de uso general y
+de dominio neutral.
+
+Una consecuencia de esta definición es que si el usuario ni siquiera *sabe qué*
+*es algo* (e.g. un grupo de hilos o un contador de un bucle, por seleccionar dos
+ejemplos arbitrarios) entonces no puede ser posible por nuestra definición de
+esencial (estamos suponiendo, por supuesto, por desgracia la posibilidad con
+algo de optimismo, que los usuarios, de hecho, conocen y comprenden el problema
+que quieren resolver).
+
+Brooks afirma [Bro86] \(y otros como Booch corroboran [Boo91]) que "la
+complejidad del software es una propiedad esencial, no accidental". Esto sugiere
+que la mayoría (al menos) de la complejidad que encontramos en los grandes
+sistemas contemporáneos es de tipo esencial.
+
+Disentimos. La complejidad no es un propiedad del software inherente (o
+esencial) de por sí (es perfectamente posible escribir software simple y sin
+embargo que siga siendo software), es más, mucha de la complejidad que vemos en
+el software existente no es esencial (para el problema). Cuando se trata de
+complejidad accidental y esencial creemos firmemente que la primera existe y que
+el objetivo de la ingeniería del software tiene que ser eliminar tanta como sea
+posible, y a la vez, ayudar con esto último.
```

Comment:
"the latter" es una forma de referirse a la segunda cosa cuando se dice "X and Y", en este caso a la complejidad esencial. Lee https://www.englishforums.com/English/FormerAndLatter/gcxc/post.htm.
En este caso: "con la segunda".